safari/environment.js

```diff
@@ -16,12 +17,12 @@ window.onunload = () => {};
 // since safari's built in extension stylesheets are treated as user stylesheets,
 // we can't inject them that way.  That makes them "user stylesheets" which would make
 // them require !important everywhere - we don't want that, so we'll inject this way instead.
-nonNull(() => document.head, 100).then(() => {
+nonNull(() => Init.headReady, 30).then(() => Init.headReady.then(() => {
```

Comment:
You can probably just use `_.defer` (same as `setTimeout(..., 0)`), since module loading is synchronous